packages/verification/src/rest/v1/callbacks/callbacks-webhook.ts

```diff
@@ -1,18 +1,59 @@
 import { VerificationRequestEvent, VerificationResultEvent } from '../../../models';
+import { SinchClientParameters, validateAuthenticationHeader } from '@sinch/sdk-client';
+import { IncomingHttpHeaders } from 'http';
 
 export type VerificationCallback = VerificationRequestEvent | VerificationResultEvent;
 
-export const parseVerificationEventNotification = (eventBody: any): VerificationCallback => {
-  if (eventBody.event) {
-    switch (eventBody.event) {
-    case 'VerificationRequestEvent':
-      return eventBody as VerificationRequestEvent;
-    case 'VerificationResultEvent':
-      return eventBody as VerificationResultEvent;
-    default:
-      throw new Error(`Unknown Verification event type: ${eventBody.event}`);
+export class CallbackWebhooks {
+
+  private sinchClientParameters: SinchClientParameters;
+
+  constructor(sinchClientParameters: SinchClientParameters) {
+    this.sinchClientParameters = sinchClientParameters;
+  }
+
+  /**
+   * Validate authorization header for callback request
+   * @param {IncomingHttpHeaders} headers - Incoming request's headers
+   * @param {string} path - Incoming request's path
+   * @param {any} body - Incoming request's body
+   * @param {string} method - Incoming request's HTTP method
+   * @return {boolean} - true if the authorization header is valid
+   */
+  public validateAuthorizationHeader(
+    headers: IncomingHttpHeaders,
+    path: string,
+    body: any,
+    method: string,
+  ): boolean {
+    if (!this.sinchClientParameters.applicationKey || !this.sinchClientParameters.applicationSecret) {
+      throw new Error('The application key and secret must be defined');
     }
+    return validateAuthenticationHeader(
+      this.sinchClientParameters.applicationKey,
+      this.sinchClientParameters.applicationSecret,
+      headers, path, body, method);
   }
-  console.log(eventBody);
-  throw new Error('Unknown Verification event');
-};
+
+  /**
+   * Reviver for a Verification Event.
+   * This method ensures the object can be treated as a Verification Event and should be called before any action is taken to manipulate the object.
+   * @param {any} eventBody - The event body containing the verification event notification.
+   * @return {VerificationCallback} - The parsed verification event object.
+   */
+  public parseVerificationEventNotification(eventBody: any): VerificationCallback {
```

Comment:
`parseEvent()` as common public interface ?

Reply:
Will be refactored in the PR#22